distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java

```diff
@@ -369,6 +370,62 @@ public void waitForRequiredBundles(String symbolicNamePrefix) throws Interrupted
         }
     }
 
+    public void waitForRequiredFeature(String featureName) throws Exception {
```

Comment:
You can extract the common code from these two methods and have them pass in a `Predicate<FeatureState>` to the common method for the test logic.

https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html